src/main/java/teammates/ui/controller/InstructorFeedbackRemindAction.java

```diff
@@ -16,11 +17,19 @@ protected ActionResult execute() {
             nextUrl = Const.ActionURIs.INSTRUCTOR_FEEDBACK_SESSIONS_PAGE;
         }
 
+        FeedbackSessionAttributes feedbackSession = logic.getFeedbackSession(feedbackSessionName, courseId);
+
         gateKeeper.verifyAccessible(
                 logic.getInstructorForGoogleId(courseId, account.googleId),
-                logic.getFeedbackSession(feedbackSessionName, courseId),
+                feedbackSession,
                 false, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
+        if (!feedbackSession.isOpened()) {
+            statusToUser.add(new StatusMessage(
+                    Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSESSIONNOTOPEN, StatusMessageColor.DANGER));
+            return createRedirectResult(nextUrl);
```

Comment:
@leeyimin If you could let me know tonight, I can have the PR ready for full review by tomorrow morning. Otherwise, I'll commit whatever I see fit, and we can take it from there. :)